src/scripts_install/@update.afni.binaries

```diff
@@ -815,11 +815,11 @@ if ( $local_package != "" ) then
    endif
 
    # get the directory of the file
-   if ( $local_package == $local_package:h ) then
```

Comment:
when would this ever be true?  Just if `$local_package` is `.`, yes?  Is that the point of this if condition, @afni-rickr ?  Nothing wrong with it

Reply:
This is true if there is no '/' in the value.